docs/install_guides/installing-openvino-yum.md

```diff
@@ -1,35 +1,19 @@
 # Install OpenVINO™ Runtime on Linux From YUM Repository {#openvino_docs_install_guides_installing_openvino_yum}
 
-
-
 @sphinxdirective
 
 .. meta::
    :description: Learn how to install OpenVINO™ Runtime on Linux operating 
-                 system, using the YUM repository, which is a recommended 
-                 installation method for C++ developers.
-
-
-With the OpenVINO™ 2023.0 release, you can install OpenVINO Runtime on Linux using the YUM repository. 
-OpenVINO™ Development Tools can be installed via PyPI only. See 
-`Installing Additional Components <#step-3-optional-install-additional-components>`__ for more information.
-
-See the `Release Notes <https://www.intel.com/content/www/us/en/developer/articles/release-notes/openvino/2023-0.html>`__ 
-for more information on updates in the latest release.
+                 system, using the YUM repository.
 
-Installing OpenVINO Runtime from YUM is recommended for C++ developers. If you are working with Python, 
-the PyPI package has everything needed for Python development and deployment on CPU and GPUs. Visit the 
-:doc:`Install OpenVINO from PyPI <openvino_docs_install_guides_installing_openvino_pip>` 
-page for instructions on how to install OpenVINO Runtime for Python using PyPI.
-
-.. warning:: 
-
-   By downloading and using this container and the included software, you agree to the terms and conditions of the 
-   `software license agreements <https://software.intel.com/content/dam/develop/external/us/en/documents/intel-openvino-license-agreements.pdf>`__.
-
-
-Prerequisites
-#############
+.. note::
+   
+   Note that the YUM distribution:
+   
+   * offers both C++ and Python APIs
```

Comment:
it DOES not include Python API, because we support obsolete system which has 3.6 python, which we don't have in our support matrix.
We need to refer somewhere that users of RPM package should install Python APIs using one of the alternative methods (conda or pip).